traveler-ios-ui/TravelerKitUI.xcodeproj/project.pbxproj

```diff
@@ -280,6 +286,8 @@
 		A996650E23442846007EBF14 /* Authenticator.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Authenticator.swift; sourceTree = "<group>"; };
 		A9D1BCFD237C5F390053C62C /* OrderPaymentViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = OrderPaymentViewController.swift; sourceTree = "<group>"; };
 		A9ECCEEA225658EA00CB5E59 /* Pass+TravelerUI.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "Pass+TravelerUI.swift"; sourceTree = "<group>"; };
+		CA0F66902394275300A104BE /* PartnerOfferingQuantityViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = PartnerOfferingQuantityViewController.swift; sourceTree = "<group>"; };
```

Comment:
Make a group for all the PartnerOffering related controllers

Reply:
Let's do that later, I don't wanna cause merge problems with the PR that did the grouping.